### PR TITLE
chore: add prettier plugin tailwindcss

### PR DESCRIPTION
**Pull request by lucianmurmurache**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Add [prettier-plugin-tailwindcss](https://www.npmjs.com/package/prettier-plugin-tailwindcss)
Formal all files to avoid complications in future PRs

### Related issue

- #770

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
     "bradlc.vscode-tailwindcss",
     "charliermarsh.ruff",
     "esbenp.prettier-vscode",
-    "heybourn.headwind",
```
Comment (andrewtavis) [inline .vscode/extensions.json:7]: Nice, thanks for this!

Comment (andrewtavis) [reply .vscode/extensions.json:7]: We need to also remove this from the readme and contributing guide as well, but I'll get those 😊 There's a dropdown with suggested extensions in each :)

File: frontend/components/card/CardTextEntry.vue [hunk 2]
```diff
@@ -20,7 +20,7 @@
         <!-- Desktop -->
         <textarea
           id="feedback"
-          class="hidden w-full px-3 py-2 overflow-y-scroll bg-transparent border border-white resize-none rounded-md md:block dark:border-white md:resize-y md:overflow-hidden"
+          class="hidden w-full resize-none overflow-y-scroll rounded-md border border-white bg-transparent px-3 py-2 md:block md:resize-y md:overflow-hidden dark:border-white"
```
Comment (andrewtavis) [inline frontend/components/card/CardTextEntry.vue:23]: Just a note to myself that we have `border-white` for each here and that this should be a custom color of some kind. I'll get to this with the text component work.

File: .vscode/extensions.json (hunk 1 above)
Comment (lucianmurmurache) [reply .vscode/extensions.json:7]: @andrewtavis I can make a quick update for the readme and contributing and update this PR. I forgot to check for any headwind mentions.

Comment (andrewtavis) [reply .vscode/extensions.json:7]: Sure thing, thanks @lucianmurmurache! I'm halfway through 🥴😅

Comment (lucianmurmurache) [reply .vscode/extensions.json:7]: ✔️

**Review by andrewtavis — APPROVED** [review]
Holy smokes was that a review 🤯 In checking through it all, I saw no cases where a class element was omitted, changed or added.

Thanks so much for this, @lucianmurmurache! Headwind was at least a good bandaid for the interim such that this review was a lot easier, but so happy to have a solution that will enforce the styles 😊 Would be great if you'd explain this a bit in the sync!

**Pull request merged**